core/arch/arm/include/arm32.h

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2016, Linaro Limited
  * Copyright (c) 2014, STMicroelectronics International N.V.
+ * SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Regardless of one may think of "All rights reserved", but if it's there it should follow the copyright statement directly. License stuff should be below.

Reply:
Agreed. I'll fix my script.